### PR TITLE
Strcdo 414

### PR DESCRIPTION
**Pull request by anatale**

### CODE REVIEW

File: components/cookbooks/meghacache/templates/default/collect_graphite_stats.rb.erb [hunk 1]
```diff
@@ -45,6 +63,7 @@ def write_to_telegraph(logfiles_path, stat_hash, telegraph_name)
 end
 
 def write_to_graphite(graphite_prefix, graphite_servers, logfiles_path, oo_cloud, oo_env, stat_hash, environment_name)
```
Comment (sergmour) [inline components/cookbooks/meghacache/templates/default/collect_graphite_stats.rb.erb]: To the best of my understanding we no longer use this write_to_graphite() method, unless I am wrong. Should we remove it? And if yes, then we can rename write_to_graphite_() to normal write_to_graphite().

File: components/cookbooks/meghacache/templates/default/collect_graphite_stats.rb.erb [hunk 2]
```diff
@@ -53,6 +72,29 @@ def write_to_graphite(graphite_prefix, graphite_servers, logfiles_path, oo_cloud
...
+      gw = GraphiteWriter.new(nil,graphite_servers,nil,nil,nil)
+      gw.open_tcp_sockets()
+      $graphiteConfigMap.each {|k,gConf|
+        puts "Trying to send data #{gConf.inspect}"
+        gw.init_helper(gConf.oo_cloud, gConf.oo_env, gConf.graphite_servers, gConf.logfiles_path, gConf.graphite_prefix, gConf.prefix_name)
+        gw.write_to_graphite(gConf.stat_hash)
+        puts "Data was sent."
+      }
+      gw.close_tcp_sockets
+    rescue => e
```
Comment (sergmour) [inline components/cookbooks/meghacache/templates/default/collect_graphite_stats.rb.erb]: Should we use`rescue Timeout::Error` ?

**Comment by sergmour** [comment]
+1 with some minor comments.
And the successful run of the script should be mute. The exceptions can be printed on stdout or stderr and piped to syslog in the cron job.

**Comment by anatale** [comment]
Yes we are going to refactor the code in the cron job after we agreed on how to redirect stdout or stderr and piped to syslog in the cron job.

File: components/cookbooks/meghacache/templates/default/collect_graphite_stats.rb.erb (hunk 2 above)
Comment (anatale) [reply components/cookbooks/meghacache/templates/default/collect_graphite_stats.rb.erb]: We are good with catch all :)

File: components/cookbooks/meghacache/templates/default/collect_graphite_stats.rb.erb (hunk 1 above)
Comment (anatale) [reply components/cookbooks/meghacache/templates/default/collect_graphite_stats.rb.erb]: Agreed. Let's remove the old function

**Pull request merged**